prediction_market_agent_tooling/markets/seer/price_manager.py

```diff
@@ -183,7 +183,7 @@ def build_probability_map(self) -> dict[OutcomeStr, Probability]:
             outcome = self.seer_market.outcomes[
                 self.seer_market.wrapped_tokens.index(outcome_token)
             ]
-            normalized_prices[outcome] = new_price
+            normalized_prices[OutcomeStr(str(outcome).strip())] = new_price
```

Comment:
```suggestion
            normalized_prices[OutcomeStr(outcome.strip())] = new_price
```

This should work 🤔 

Wouldn't be a big deal, just if we ever change type of `outcome` to something else, mypy wouldn't complain here as almost anything can be stringified.

Reply:
Agree fixed.

Reply:
done